dvc/rwlock.py

```diff
@@ -4,16 +4,21 @@
 from collections import defaultdict
 from contextlib import contextmanager
 
-from voluptuous import Schema, Invalid
+from voluptuous import Schema, Invalid, Required
 from funcy.py3 import lmap, lfilter, lmapcat
 
 from .exceptions import DvcException
 from .lock import LockError
 from .utils import relpath
 
-INFO_SCHEMA = {"pid": int, "cmd": str}
+INFO_SCHEMA = Schema({Required("pid"): int, Required("cmd"): str})
```

Comment:
No need to wrap with `Schema`, as it is already present in `SCHEMA` down below.
```suggestion
INFO_SCHEMA = {Required("pid"): int, Required("cmd"): str}
```

Reply:
Indeed :+1: